samples/matter/template/README.rst

```diff
@@ -126,44 +126,56 @@ For example:
 
 .. matter_template_build_with_tfm_end
 
+Advanced configuration options
+==============================
+
+This section describes other configuration options for the sample.
+
 Device Firmware Upgrade support
-===============================
+-------------------------------
 
 .. |Bluetooth| replace:: Bluetooth
 
-.. include:: ../lock/README.rst
-    :start-after: matter_door_lock_sample_build_with_dfu_start
-    :end-before: matter_door_lock_sample_build_with_dfu_end
+.. toggle::
 
-Alternatively, for the nRF54L15 DK, the DFU can be configured to only use the internal MRAM for storage.
-This means that both the currently running firmware and the new firmware to be updated will be stored within the device's internal flash memory.
-This configuration is enabled by default for the :ref:`debug configuration <matter_template_custom_configs>`.
+   .. include:: ../lock/README.rst
+       :start-after: matter_door_lock_sample_build_with_dfu_start
+       :end-before: matter_door_lock_sample_build_with_dfu_end
 
-The following is an example command to build the sample on the nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal MRAM only:
+   Alternatively, for the nRF54L15 DK, the DFU can be configured to only use the internal MRAM for storage.
```

Comment:
Probably old issue:

```suggestion
   Alternatively, for the nRF54L15 DK, the DFU can be configured to only use the internal RRAM for storage.
```

Reply:
How about line 149, should it be RRAM as well?